app/components/filters/exclude_cancelled.rb

```diff
@@ -0,0 +1,9 @@
+module Filters
```

Comment:
I would disagree with this as I see the ETL code almost as an extractable gem. As such I would not want to reuse it's namespace.

I could see a point to changing Filters to ETLFilter or similar which shows the link without reuse of the namespace.